dawgz/schedulers.py

```diff
@@ -522,10 +522,15 @@ async def exec(self, job: Job) -> Any:
                 ])
             )
 
+        if self.interpreter is not None:
+            interpreter = self.interpreter
+        else:
+            interpreter = job.interpreter
```

Comment:
I would prefer the opposite actually

```suggestion
        if job.interpreter is None:
            interpreter = self.interpreter
        else:
            interpreter = job.interpreter
```

Job settings should have precedence over scheduler settings (see L453).

Reply:
Yeah, makes more sense I agree.